apps/shared/avifjpeg.c

```diff
@@ -996,11 +996,11 @@ static avifBool avifJPEGReadInternal(FILE * f,
                     goto cleanup;
                 }
 
-                if (marker->data_length - AVIF_JPEG_EXIF_HEADER_LENGTH > imageSizeLimit) {
+                if (marker->data_length - AVIF_JPEG_EXIF_HEADER_LENGTH > sizeLimit) {
```

Comment:
In line 1001 and line 1079, change "px" to "bytes" in the error messages.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2048